src/utils/ManagerUtils.ts

```diff
@@ -0,0 +1,13 @@
+import GameManager from "../model/game/GameManager";
+import ManagerSettings from "../r2mm/manager/ManagerSettings";
+
+
+/**
+ * Return default game selection needed to skip the game selection screen.
+ */
+export const getDefaults = (settings: ManagerSettings) {
+    const globals = settings.getContext().global;
+    const defaultGame = GameManager.findByFolderName(globals.defaultGame);
+    const defaultPlat = defaultGame?.storePlatformMetadata.find(x => x.storePlatform === globals.defaultStore);
+    return {defaultGame, defaultPlatform: defaultPlat?.storePlatform};
```

Comment:
Could you remove the optional chaining operator for now?

Reply:
Replaced two occurrences of optional chaining with good old ternary operator.